templates/documentation/delivery-analytics/private-video-delivery-web-browser.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: "Private video delivery with api.video players"
+title: Private video playback and delivery
+slug: private-video-delivery-web-browser
```

Comment:
`slug` can be removed, as confirmed by Doctave [here](https://api-video.slack.com/archives/C05DTNR3QDS/p1697618392282579)

Reply:
Done ✅